truffe2/accounting_tools/templates/accounting_tools/expenseclaim/pdf.html

```diff
@@ -37,13 +37,22 @@ <h1 style="text-align: center; font-size: 30px; margin-top:-150px;">{% trans "No
          <tr><td>{% trans "Nombre de justificatifs : " %}</td><td>{{object.nb_proofs}}</td><td colspan="2"></td></tr>
          <tr><td colspan="4">&nbsp;</td></tr>
 
-        <tr><td colspan="2"><b>{% trans "Visa de la personne : " %}</b></td><td colspan="2" style="border-bottom: 1px black solid;"></td></tr>
-        <tr><td colspan="4">&nbsp;</td></tr><tr><td colspan="4">&nbsp;</td></tr>
+      <tr>
+            <td colspan="3"><b>{% trans "Visa de l'unité : " %}</b></td>
+            <td colspan="4" style="border-bottom: 1px black solid;">{% if object.status == '2_agep_validable' or object.status == '3_agep_sig1' or object.status == '3_agep_sig2' or object.status == '4_accountable' or object.status == '5_in_accounting' or object.status == '6_archived' %}{% if object.get_unit_signer %}<img src="{{MEDIA_ROOT}}img/valid-circle-icon.png" style="margin-right: 8px;" height="13" width="13">&nbsp;&nbsp;{{ object.get_unit_signer.first_name }}  {{ object.get_unit_signer.last_name }}{% endif %}{% endif %}</td>
```

Comment:
Pourquoi le if à chaque fois ?

Reply:
Seems that I can't do better : https://docs.djangoproject.com/en/dev/ref/templates/builtins/#complex-expressions

Reply:
Or that i didn't undesrtand you point

Reply:
En fait c'est plus le coup de recopier les trucs de status, t'arrives pas à simplifier en mettant ça dans le modèle avec une property ?